src/routes/+layout.svelte

```diff
@@ -3,10 +3,10 @@
   import { Footer } from '@frequency-chain/style-guide';
 </script>
 
-<div class="freq-container">
-  <Header />
-</div>
-<div class="max-w-screen-xl">
+<Header />
+<div class="w-full">
   <slot />
 </div>
-<Footer type="dark" class="w-full border-grayBorder border-opacity-25 bg-navy vertical-lg:w-[100%]" />
+<div class="w-full bg-navy px-[5%] lg:px-[20%]">
+  <Footer type="dark" class="w-full border-grayBorder border-opacity-25 vertical-lg:w-[100%]" />
```

Comment:
Allows the footer to have a full width background color along with the desired padding from the desgin